model/config/config.yaml

```diff
@@ -13,10 +13,50 @@ logger:
         handlers: [console]
 
 IGPreprocessor:
-    raw_data_path: ./.cache/data/instagram/ig_sample.tar.gz
-    raw_data_url: https://tweakstory.stefanmesken.info/data/ig_sample.tar.gz
-    raw_data_directory: ig_sample
+    raw_data_path: "./.cache/data/instagram/ig_sample.tar.gz"
+    raw_data_url: "https://tweakstory.stefanmesken.info/data/ig_sample.tar.gz"
+    raw_data_directory: "ig_sample"
     raw_data_group_names:
-        insta-caption-train.json: train
-        insta-caption-test1.json: eval
-        insta-caption-test2.json: test 
+        insta-caption-train.json: "train"
+        insta-caption-test1.json: "eval"
+        insta-caption-test2.json: "test"
+
+Trainer:
+    pad_token_id: 0
+    unk_token_id: 0
+    batch_size: 16
+    max_epochs: 100
+    device_name: "cpu"
+    dataloader_num_workers: 2
+
+    IGDataset:
+        cache_location: "./.cache/data/instagram/"
+        dataset_name: "ig_sample"
+        split: "train"
+        image_size: 320
+        crop_size: 320
+        image_normalization_mean:
+        - 0.485
+        - 0.456
+        - 0.406
+        image_normalization_std:
+        - 0.229
+        - 0.224
+        - 0.225
+
+    ShowAttendTell:
+        ResnetEncoder:
+            input_height: 320
+            input_width: 320
+            hidden_dim: 300
+
+        AttentionLSTMDecoder:
+            hidden_dimension: 300
+            embedding_dimension: 300
+            vocabulary_size: 100000
```

Comment:
is this the minimum vocab size of our IG dataset?

Reply:
It's the vocabulary size of the pretrained embedding. We'll likely use a different value here. But determining the best choice is left for another PR.